services/mlmodel/mlmodel.go

```diff
@@ -28,9 +28,22 @@ func init() {
 // Service defines the ML Model interface, which takes a map of inputs, runs it through
 // an inference engine, and creates a map of outputs. Metadata is necessary in order to build
 // the struct that will decode that map[string]interface{} correctly.
+//
+// Infer example:
+//
+//	input_tensors := ml.Tensors{"0": tensor.New(tensor.WithShape(1, 2, 3), tensor.WithBacking(6))}
```

Comment:
I don't think WithBacking works like that (I could be wrong tbh). This way below definitely works and is more clear.  

`input_tensors := ml.Tensors{"0": tensor.New(tensor.WithShape(1, 2, 3), tensor.WithBacking([]int{1, 2, 3, 4, 5, 6}) )}`